packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +75,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.srcdoc) {
+      if (frameOptions.iframeAttrs === null
```

Comment:
since options is not required prop it can be undefined as well so accessing property from undefined object (frameOptions.iframeAttrs) will throw up errors. Either we can use default props to set value for options prop to be `{}` so that it does not throw up an error while accessing the property `iframeAttrs`

Reply:
There is a check for `frameOptions.iframeAttrs === undefined`.

Reply:
what happens when `frameOptions` is undefined ?

Reply:
Yeah, on line #84, an empty object is initialized and assigned to `frameOptions.iframeAttrs`.